src/main/java/aichan/task/Task.java

```diff
@@ -0,0 +1,100 @@
+package aichan.task;
+
+import aichan.AiChanException;
+
+/**
+ * Represents a task.
+ */
+public class Task {
+    private String taskName;
+    private boolean isDone;
+
+    /**
+     * Constructs a task object.
+     * Initializes description and set isDone to false.
+     *
+     * @param taskName Description of the task.
+     */
+    public Task(String taskName) {
+        this.taskName = taskName;
+        this.isDone = false;
+    }
+
+    /**
+     * Converts a line in the file to its task.
+     *
+     * @param str String of one line in a file.
+     * @return The corresponding task.
+     * @throws AiChanException If the line has incorrect format.
+     */
+    public static Task stringToTask(String str) throws AiChanException {

```

Comment:
Perhaps you could use verbs to name the method? Maybe something like `getTaskFromCommand(String str)`

```suggestion
    public static Task getTaskFromCommand(String str) throws AiChanException {
```